data_utils/connect.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy import MetaData, Engine
+from sqlalchemy.ext.automap import automap_base, AutomapBase
+
+
+def get_automap_base_with_views(engine: Engine, schema: str) -> AutomapBase:
+    '''
+    Returns an AutomapBase object that also provides access to views of a schema
+
+    Parameters
+    ----------
+    engine : sqlalchemy.Engine
+        Engine that is already connected to a database
+    schema : str
+        Name of schema in the database for which we want to get an AutomapBase object
+
+    Returns
+    -------
+    AutomapBase
+        A declarative automap base for the schema of the database
+
+    Examples
+    --------
+    >>> from sqlalchemy import create_engine
+    >>> engine = create_engine('postgresql://user:password@localhost:5432/dbname'))
+    >>> Base = get_automap_base_with_views(engine=engine, schema='orca')
+    >>> print(type(Base))
+    <class 'sqlalchemy.ext.automap.AutomapBase'>
+    '''
+    metadata = MetaData()
+    metadata.reflect(bind=engine, views=True, schema=schema)
+    Base: AutomapBase = automap_base(metadata=metadata)
+    Base.prepare()
+    return Base
+
+if __name__=='__main__':
+    import os
+
+    from dotenv import load_dotenv
+    from sqlalchemy import create_engine
+
+    path_env = os.path.join(os.path.dirname(os.getcwd()), '.env')
+    load_dotenv(dotenv_path=path_env)
+    POSTGRES_URL = 'POSTGRES_URL'
+    SCHEMA = 'orca'
+
+    engine = create_engine(os.getenv(POSTGRES_URL))
+    Base = get_automap_base_with_views(engine=engine, schema=SCHEMA)
```

Comment:
I think it'd be nice to put this in its own function that returns the engine. Then you can just call that function to get the engine, which can be passed directly to your `get_automap_base_with_views` function.

Reply:
I see. 
Should the function include the logic of extracting the POSTGRES_URL environment variable? I am not sure how that would work, or whether that's safe.

Reply:
I see no reason for it not to, I don't think there's anything unsafe about it unless I'm missing something, but it might be nice to issue a helpful error message if it cannot find the required environment variable. Also make sure there are docs on how to properly create the dotenv file.